certbot-nginx/certbot_nginx/parser.py

```diff
@@ -344,6 +312,16 @@ def add_server_directives(self, vhost, directives, replace):
         except errors.MisconfigurationError as err:
             raise errors.MisconfigurationError("Problem in %s: %s" % (filename, str(err)))
 
+def _parse_flat_file(ssl_options):
```

Comment:
nit: If you want to make this function be specific to our TLS options file, I'd update the function name to reflect that. I see a benefit in making this more general as we continue to use more include directives though so perhaps you should update variable names and error messages to make it more useful for parsing included files in the future.